indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -381,11 +409,16 @@ private void flushIndexToContextAndClose(BytesWritable key, IncrementalIndex ind
         context.progress();
         Row row = rows.next();
         InputRow inputRow = getInputRowFromRow(row, dimensions);
-        // reportParseExceptions is true as any unparseable data is already handled by the mapper.
-        context.write(
-            key,
-            new BytesWritable(InputRowSerde.toBytes(inputRow, combiningAggs, true))
-        );
+
+        byte[] serializedRow = InputRowSerde.toBytes(typeHelperMap, inputRow, combiningAggs, true);
+
+        if (serializedRow != null) {
+          // reportParseExceptions is true as any unparseable data is already handled by the mapper.
```

Comment:
This comment is attached to the wrong line. It should go with toBytes.

Reply:
Moved comment